### PR TITLE
[T4A3][W10-B2] Moon Byunghun

### PR DESCRIPTION
**Pull request by MoonByunghun**

### CODE REVIEW

**Comment by MoonByunghun** [comment]
Ready for review.

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact in the address book.
+ * Guarantees: immutable;
+ */
+
+public class Contact {
+    public final String value;
+    private boolean isPrivate;
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/data/person/Contact.java:10]: Should it be private here? Will it be inherited?

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+    
+    @Override
+    public String toString(){
+        return value;
+    }
+    
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/data/person/Contact.java:24]: May not need comments for standard overriding.

**Comment by OuyangDanwen** [comment]
@MoonByunghun It is important to understand the difference among attribute types, i.e public, protected and private, so that you use the right amount of exposure. 
Please close the PR after reading the comments.

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact in the address book.
+ * Guarantees: immutable;
+ */
+
+public class Contact {
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/data/person/Contact.java:8]: As discussed in tutorial, it is better to use abstract class here.

**Pull request closed without merging**